multihost_test/bz_automation/test_automation.py

```diff
@@ -127,3 +127,31 @@ def test_2091062(self, multihost, create_localusers):
         client.logout()
         assert "pam_motd: error scanning directory" not in \
                execute_cmd(multihost, "cat /var/log/secure").stdout_text
+    def test_pam_faillock_audit(self, multihost, create_localusers, bkp_pam_config):
+        """
+        :title: Pam_faillock audit events duplicate uid.
+        :id: c62fda84-401b-11ee-90bd-845cf3eff344
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=2231556
+        :setup:
+            1. Enable pam_faillock in the PAM stack (either with authselect "authselect
+                enable-feature with-faillock" or manually)
+            2. Modify pam_faillock "deny" option in /etc/security/faillock.conf to
+                lock the user at the first attempt (deny=1)
+        :steps:
+            1. Authenticate as user and input an incorrect password
+            2. Check that audit file (/var/log/audit/audit.log) contains the correct format.
+        :expectedresults:
+            1. Authentication should fail.
+            2. Pam_faillock audit must display messages with the correct format:
+                op=pam_faillock suid=UID. Where UID is the ID of the user trying to authenticate.
+        """
+        client = multihost.client[0]
+        uid = client.run_command("id -u local_anuj").stdout_text.split()[0]
+        client.run_command("authselect enable-feature with-faillock")
```

Comment:
Now that I see it written I think the command is `authselect enable-feature with-faillock sssd`, but if you already run the test case and it's working disregard this comment